### DISCUSSION TITLE
Work Account got suspended all of a sudden

### DISCUSSION THREAD

**Discussion by iArunava** [community/community#75833] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi. My work account got suspended yesterday. @arc-aviso 
It was hacked few days back and 2-3 weird respositories with no code got created. 
I changed my password and enabled 2FA and made sure there is one active session.

I think i shouldv deleted those repos. Had some work. Didnt get around to it. Now they suspended the account. I opened a ticket but i dont think anybody will respond there. 

Can somebody please help me out here.
[Redacted due to PII]

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as the Community is unable to assist with flagged accounts :) If you've already opened a ticket, it will be answered in the order it was received based on the resources of our team. 

In the future please do not post links to tickets to avoid sharing any personal information.
